packages/arb-upgrades/index.ts

```diff
@@ -0,0 +1,361 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types/runtime'
+import { writeFileSync, readFileSync } from 'fs'
+import childProcess from 'child_process'
+import {
+  QueuedUpdates,
+  CurrentDeployments,
+  ContractNames,
+  CurrentDeployment,
+  QueuedUpdate,
+  isBeacon,
+} from './types'
+
+// import {
+//   getStorageLayout,
+//   assertUpgradeSafe,
+//   getContractVersion,
+//   assertStorageUpgradeSafe,
+//   solcInputOutputDecoder,
+//   validate,
+//   RunValidation,
+// } from '@openzeppelin/upgrades-core'
+
+const adminSlot =
```

Comment:
we should probably add a comment that references where this value comes from

ie https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.0/contracts/proxy/TransparentUpgradeableProxy.sol#L48

same for the beacon and implementation slot